common/metrics/src/lib.rs

```diff
@@ -1,21 +1,82 @@
-use std::sync::atomic::AtomicI64;
-use std::sync::atomic::{AtomicU64, Ordering};
-
-use chrono::Utc;
 use jemalloc_ctl::stats::{active, active_mib, allocated, allocated_mib, resident, resident_mib};
 use jemalloc_ctl::{epoch, epoch_mib};
 use json::object;
 use lazy_static::lazy_static;
 use log::info;
-use std::thread::sleep;
-use std::time::Duration;
+use prometheus::{
+    exponential_buckets, register_histogram, register_histogram_vec, register_int_counter,
+    register_int_counter_vec, register_int_gauge, Histogram, HistogramVec, IntCounter,
+    IntCounterVec, IntGauge,
+};
+use std::time::{Duration, Instant};
+use tokio::time::sleep;
 
 lazy_static! {
     static ref METRICS: Metrics = Metrics::new();
+    static ref FS_EVENTS: IntCounterVec = register_int_counter_vec!(
+        "logdna_fs_events",
```

Comment:
I'm starting to think that `logdna_agent_` is a better prefix. Having `company_product_` prefixes avoids collisions in the future.

Reply:
I agree on that